src/styles/theme/themeConfig.ts

```diff
@@ -0,0 +1,203 @@
+// Base colors - updates should match css vars!
+const PURE_WHITE = "#ffffff";
+const DARK = "#141219";
+const DARK_THREE = "#25222e";
+const DARK_FOUR = "#2f2a3c";
+const OFF_WHITE = "#f7f7f7";
+const WHITE_SIX = "#e7e7e7";
+const WHITE_THREE = "#d8d8d8";
+const BABY_PURPLE = "#b59ff6";
+const LIGHT_PURPLE = "#e7e4f2";
+const CHARCOAL_GRAY = "#3b3649";
+const ALLEN_PURPLE = "#8d87aa";
+const GRAYISH_BROWN = "#4a4a4a";
+const BLUE = "#0094FF";
+const WARM_GRAY = "#979797";
+const TRANSPARENT_WHITE = "rgba(255, 255, 255, 0.65)";
+const TRANSPARENT = "transparent";
+
+const APP_BG_COLOR = OFF_WHITE;
+const TABS_INTERACTION_COLOR = DARK_FOUR;
+const SLIDER_COLOR = WHITE_THREE;
+const CHECKBOX_BG_COLOR = WHITE_SIX;
+
+// Light Theme Tokens (Global defaults, Modal, Radio)
+const lightTheme = {
```

Comment:
I've gotten confused working with this repo in the past by assuming that "light theme" and "dark theme" were different modes for the whole UI that could be switched between dynamically, rather than design concepts that statically apply to different UI elements (light theme for landing page, dark theme for viewer controls, etc.). Consider adding a few extra words to this comment to help out confused devs like my past self.

Also: even given that these "themes" apply to different UI elements, I'm a bit surprised that more effort isn't taken to unify the shapes of these objects, but that's out of scope for this PR anyways.

Reply:
Hate to say it but I think I have found an even better shiny new way of organizing these variables, so this may code might not last that long. This uses styled-components so is pending some buy-in from reviewer and Megan but I like it and its pointed at this branch. It gets rid of the light and dark theme naming you see here.
https://github.com/simularium/simularium-website/pull/616

I agree that the light/dark theme is confusing, and I am also hoping to get rid of the light-theme.css file in favor of some CSS-in-JS and actually using themes for theming when things are called themes.